### DISCUSSION TITLE
How can I change the style of the inputPrompt or infoDialog?

### DISCUSSION THREAD

**Discussion by planerfolg** [chhoumann/quickadd#619] [Q&A]
Hi, 
I would like to change the color of the dialog boxes and the text prompts. In the css I could not identify the correct part. Does anyone have an idea? Thanks in advance.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

QuickAdd's prompts carry their own CSS classes, so a CSS snippet (Settings -> Appearance -> CSS snippets) can restyle them:

- all QuickAdd modals: `.quickAddModal`
- the input prompt: `.qaInputPrompt`
- the large multiline prompt: `.qaWideInputPrompt`
- the yes/no prompt: `.qaYesNoPrompt`

For example:

```css
.quickAddModal .modal {
  background-color: #1e2a38;
}

.qaInputPrompt input {
  color: hotpink;
}
```

One honest caveat: the info dialog (`infoDialog` from the API) currently has no QuickAdd-specific class, so you can only reach it through Obsidian's generic `.modal` selectors.

If a selector doesn't take effect in your theme, open the developer tools (Ctrl/Cmd+Shift+I), inspect the dialog, and check which classes the element actually carries.
